packages/fast-development-site-react/src/components/site/dev-tools-code-preview-style.ts

```diff
@@ -0,0 +1,201 @@
+export default {
+    "code[class*=\"language-\"]": {
+        "color": "#464646",
+        "fontFamily": "\"Consolas\", \"Bitstream Vera Sans Mono\", \"Courier New\", Courier, monospace",
+        "direction": "ltr",
+        "textAlign": "left",
+        "whiteSpace": "pre",
+        "wordSpacing": "normal",
+        "wordBreak": "normal",
+        "fontSize": "0.95em",
+        "lineHeight": "1.2em",
+        "MozTabSize": "4",
+        "OTabSize": "4",
+        "tabSize": "4",
+        "WebkitHyphens": "none",
+        "MozHyphens": "none",
+        "msHyphens": "none",
+        "hyphens": "none"
+    },
+    "pre[class*=\"language-\"]": {
+        "color": "#464646",
+        "fontFamily": "\"Consolas\", \"Bitstream Vera Sans Mono\", \"Courier New\", Courier, monospace",
+        "direction": "ltr",
+        "textAlign": "left",
+        "whiteSpace": "pre",
+        "wordSpacing": "normal",
+        "wordBreak": "normal",
+        "fontSize": "0.95em",
+        "lineHeight": "1.2em",
+        "MozTabSize": "4",
+        "OTabSize": "4",
+        "tabSize": "4",
+        "WebkitHyphens": "none",
+        "MozHyphens": "none",
+        "msHyphens": "none",
+        "hyphens": "none",
+        "padding": "1em",
+        "margin": ".5em 0",
+        "overflow": "auto",
+        "backgroundColor": "white"
+    },
+    "pre[class*=\"language-\"]::-moz-selection": {
+        "background": "#C1DEF1"
+    },
+    "pre[class*=\"language-\"] ::-moz-selection": {
+        "background": "#C1DEF1"
+    },
+    "code[class*=\"language-\"]::-moz-selection": {
+        "background": "#C1DEF1"
+    },
+    "code[class*=\"language-\"] ::-moz-selection": {
+        "background": "#C1DEF1"
+    },
+    "pre[class*=\"language-\"]::selection": {
+        "background": "#C1DEF1"
+    },
+    "pre[class*=\"language-\"] ::selection": {
+        "background": "#C1DEF1"
+    },
+    "code[class*=\"language-\"]::selection": {
+        "background": "#C1DEF1"
+    },
+    "code[class*=\"language-\"] ::selection": {
+        "background": "#C1DEF1"
+    },

```

Comment:
On lines 42 - 63 it appears that each pair (pre/code) is duplicated.

Reply:
Let's remove these duplications above. I can't find any reason we would need the duplication here.